lib/Echidna/Types/Campaign.hs

```diff
@@ -211,9 +211,14 @@ defaultSymExecAskSMTIters :: Integer
 defaultSymExecAskSMTIters = 1
 
 -- | Get number of fuzzing workers (doesn't include sym exec worker)
--- Defaults to 1 if set to Nothing
+-- Defaults to (N-1) if set to Nothing, where N is Haskell's -N value,
+-- usually the number of cores, clamped between 1 and 3.
 getNFuzzWorkers :: CampaignConf -> Int
-getNFuzzWorkers conf = fromIntegral (fromMaybe 1 (conf.workers))
+getNFuzzWorkers conf = maybe defaultN fromIntegral conf.workers
+  where
+    n = numCapabilities
+    maxN = max 1 (n - 1) -- one dedicated thread for shrinking
+    defaultN = min 3 maxN -- capped at 3 by default
```

Comment:
why do we cap at 3?

Reply:
@ggrieco-tob suggested the value

Reply:
it's a reasonable number for default, we can increase if we want later.

Reply:
I switched it to 4, considering I dropped the `-1`